README.md

```diff
@@ -0,0 +1,82 @@
+# DSS Safebox
+
+A safebox for digital assets.
+
+- [Overview](#overview)
+- [Deployment](#deployment)
+- [Usage](#usage)
+  - [Deposit assets](#deposit-assets)
+  - [Withdraw assets](#withdraw-assets)
+  - [Change the `recipient` address](#change-the-recipient-address)
+
+## Overview
+
+There are 3 "roles" in this contract:
+
+1. `owner`: has full control of how much and when it can send assets to `recipient`.
+2. `custodian`: approves changes in the `recipient` address made by the `owner`.
```

Comment:
mmm namewise -  this entity is not really storing/custodying the tokens, it can only block changing the recipient.

Reply:
It has to be named like this for legal reasons.